src/hct_mis_api/one_time_scripts/remove_migrated_data_is_original.py

```diff
@@ -6,15 +6,26 @@ def remove_migrated_data_is_original() -> None:
 
     for model in all_models:
         if hasattr(model, "is_original"):
-            print(f"Removing objects from {model.__name__} that have the 'is_original' field.")
             if model.__name__ == "GrievanceTicket":
-                # or maybe try except AttributeError:
                 # 'GrievanceTicket' has no attribute 'all_objects'
-                queryset = model.default_for_migrations_fix.filter(is_original=True)
+                queryset_all = model.default_for_migrations_fix.all()
+                queryset_is_original = queryset_all.filter(is_original=True)
             elif model.__name__ in ["HouseholdSelection", "EntitlementCard", "Feedback", "Message"]:
-                queryset = model.original_and_repr_objects.filter(is_original=True)
+                queryset_all = model.original_and_repr_objects.all()
+                queryset_is_original = queryset_all.filter(is_original=True)
             else:
-                queryset = model.all_objects.filter(is_original=True)
+                queryset_all = model.all_objects.all()
+                queryset_is_original = queryset_all.filter(is_original=True)
 
-            count, _ = queryset.delete()
-            print(f"Deleted {count} {model.__name__} objects.\n")
+            print(
+                f"*** {model.__name__} All objects: {queryset_all.count()}. "
+                f"Removing objects with 'is_original=True': {queryset_is_original.count()}"
+            )
+
+            try:
+                count, _ = queryset_is_original.delete(soft=False)
+            #  TypeError: QuerySet.delete() got an unexpected keyword argument 'soft'
```

Comment:
It will fail for all, because you need to iterate over them; it is not possible to call `soft=` on a queryset

Reply:
just call qs.delete() in all cases

Reply:
you can just use delete(), beacuse all of above managers all_objects/default_for_migrations/original_and_repr_objects are not soft deletable, right?

Reply:
yes @MarekBiczysko you are right! that's why test passed

Reply:
But for the ones using it, you are taking `all_objects` so it actually works in this case :)

Reply:
so for `EntitlementCard` it's `original_and_repr_objects = models.Manager()` :ok_hand:
`Feedback` >> `original_and_repr_objects = OriginalAndRepresentationsManager() > models.Manager` :ok_hand:
`Message` >> `original_and_repr_objects = OriginalAndRepresentationsManager() > models.Manager` :ok_hand:
`HouseholdSelection` >> `original_and_repr_objects = models.Manager()` :ok_hand:

Reply:
hope everything have to be okay 😄